tests/test_parser.py

```diff
@@ -694,3 +694,15 @@ def test_expand_concatenated_variables(self, dfparser):
             LABEL component="$NAME$VER"
         """)
         assert dfparser.labels['component'] == 'name1'
+
+    def test_multiline(self, dfparser):
+        dfparser.content = dedent("""
+            LABEL a=b c d e \
```

Comment:
What happens if there are multiple spaces between characters?

Reply:
oh my, someone would actually do that?

anyway, `docker` fails to parse such dockerfile:

```
$ docker build .
Sending build context to Docker daemon 4.096 kB
Error response from daemon: Syntax error - can't find = in "c". Must be of the form: name=value
$ cat Dockerfile
FROM scratch
LABEL a=b   c
```